src/app/features/stacks-high-fee-warning/stacks-high-fee-dialog.tsx

```diff
@@ -60,14 +54,13 @@ export function HighFeeDialog({ learnMoreUrl, isShowing = false }: HighFeeDialog
         <HStack>
           <ErrorIcon color="red.action-primary-default" width="md" />
           <Title>
-            Are you sure you want to pay {values.fee} {values.feeCurrency} in fees for this
-            transaction?
+            Are you sure you want to pay {(values as any).fee} {(values as any).feeCurrency} in fees
```

Comment:
Maybe we can add types for `values` as `StacksSendFormValues` / `StacksTransactionFormValues` and avoid the `any` cast